src/potential.exploiters.ts

```diff
@@ -0,0 +1,39 @@
+import { Finding, getEthersProvider, HandleTransaction, TransactionEvent } from "forta-agent";
+import { Trace } from "forta-agent/dist/sdk/trace";
+import LRU from "lru-cache";
+import { FindingGenerator } from "./utils";
+import ethers from "ethers";
+
+export default function providePotentialExploiterHandler(
+  provider: ethers.providers.JsonRpcProvider,
+  findingGenerator: FindingGenerator,
+  tornadoAddresses: string[],
+  withdrawalEventAbi: string,
+  suspiciousHexStrings: string[]
+): HandleTransaction {
+  const tornadoFundedAddresses: LRU<string, boolean> = new LRU({ max: 1000000 });
+  return async (txEvent: TransactionEvent): Promise<Finding[]> => {
+    const findings: Finding[] = [];
+    let hex_present: string[];
+    const tornadoWithdrawalsEvents = tornadoAddresses
+      .map((tornadoAddress: string) => txEvent.filterLog(withdrawalEventAbi, tornadoAddress))
+      .flat();
+
+    tornadoWithdrawalsEvents.forEach((tornadoWithdrawal) => {
+      tornadoFundedAddresses.set(tornadoWithdrawal.args["to"].toLowerCase(), true);
+    });
+
+    for (let i = 0; i < txEvent.traces.length; ++i) {
+      const trace: Trace = txEvent.traces[i];
+      if (trace.type === "create" && tornadoFundedAddresses.get(trace.action.from)) {
+        const contractAddress: string = txEvent.contractAddress as string;
+        const bytecode: string = await provider.getCode(contractAddress);
+        hex_present = suspiciousHexStrings.filter((el) => bytecode.includes(el));
+        if (hex_present.length > 0) {
+          findings.push(findingGenerator({ from: trace.action.from, contractAddress, suspiciousHexValues: hex_present }));
```

Comment:
`metedata` only allows strings
```suggestion
          findings.push(findingGenerator({ from: trace.action.from, contractAddress, suspiciousHexValues: hex_present.toString() }));
```